core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/huggingFace/HuggingFaceIrisLogisticRegressionOpDesc.scala

```diff
@@ -0,0 +1,120 @@
+package edu.uci.ics.texera.workflow.operators.huggingFace
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+class HuggingFaceIrisLogisticRegressionOpDesc extends PythonOperatorDescriptor {
+
+  @JsonProperty(value = "petalLengthCmAttribute", required = true)
+  @JsonPropertyDescription("column in your dataset corresponding to PetalLengthCm")
+  @AutofillAttributeName
+  var petalLengthCmAttribute: String = _
+
+  @JsonProperty(value = "petalWidthCmAttribute", required = true)
+  @JsonPropertyDescription("column in your dataset corresponding to PetalWidthCm")
+  @AutofillAttributeName
+  var petalWidthCmAttribute: String = _
+
+  @JsonProperty(
+    value = "prediction class name",
+    required = true,
+    defaultValue = "Species_prediction"
+  )
+  @JsonPropertyDescription("column name for the predicted class of species")
+  var predictionClassName: String = _
+
+  @JsonProperty(
+    value = "prediction probability name",
+    required = true,
+    defaultValue = "Species_probability"
+  )
+  @JsonPropertyDescription("column name for the prediction's probability of being a Iris-setosa")
+  var predictionProbabilityName: String = _
+
+  /**
+    * This method is to be implemented to generate the actual Python source code
+    * based on operators predicates.
+    *
+    * @return a String representation of the executable Python source code.
+    */
```

Comment:
this scala-doc is not relevant (it is for the base). please modify it to describe the actual behavior of this override function.

Reply:
removed